src/main/java/configure/AppConfigure.java

```diff
@@ -0,0 +1,44 @@
+package configure;
+
+import annotation.Bean;
+import controller.Controller;
+import controller.UserController;
+import servlet.DispatcherServlet;
+import session.SessionManager;
+import util.HttpRequestParser;
+import view.View;
+import view.ViewResolver;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+
+// TODO : 현재는 싱글톤 처럼 동작하지는 않는다.
+public class AppConfigure {
```

Comment:
```suggestion
public class AppConfiguration{
```
클래스 이름은 명사, 메소드는 동사가 좋습니다.